services/app-web/src/components/FileUpload/FileItem/FileItem.tsx

```diff
@@ -120,16 +120,27 @@ export const FileItem = ({
         retryItem(item)
     }
 
+    let statusValue = "";
+    if (isLoading) {
+        statusValue = "uploading";
+    } else if (isScanning) {
+        statusValue = "scanning for viruses"
+    } else if (hasDuplicateNameError || hasScanningError || hasUploadError || hasUnexpectedError) {
+        statusValue = "error"
+    }
+
     return (
         <>
             <div className={styles.fileItemText}>
-                <img
-                    id={item.id}
-                    data-testid="file-input-preview-image"
-                    src={SPACER_GIF}
-                    alt=""
-                    className={imageClasses}
-                />
+                <div role="progressbar" aria-valuetext={statusValue} aria-label={`Status of file ${name}`}>
```

Comment:
this use of progressbar is 💅 I like how it maps to the visual UI we already have established